core/lib/dal/src/tee_proof_generation_dal.rs

```diff
@@ -249,23 +249,21 @@ impl TeeProofGenerationDal<'_, '_> {
                 tp.signature,
                 tp.proof,
                 tp.updated_at,
+                tp.status,
                 ta.attestation
             FROM
                 tee_proof_generation_details tp
             LEFT JOIN
                 tee_attestations ta ON tp.pubkey = ta.pubkey
             WHERE
                 tp.l1_batch_number = $1
-                AND tp.status = $2
                 {}
             ORDER BY tp.l1_batch_number ASC, tp.tee_type ASC
             "#,
-            tee_type.map_or_else(String::new, |_| "AND tp.tee_type = $3".to_string())
+            tee_type.map_or_else(String::new, |_| "AND tp.tee_type = $2".to_string())
```

Comment:
Off-topic (?): It's possible to use [the `match_query_as` macro](https://github.com/matter-labs/zksync-era/blob/a10c4baa312f26ebac2a10115fb7bd314d18b9c1/core/lib/db_connection/src/macro_utils.rs#L84) here to make the produced queries verified in compile time. There are examples of its usage in other modules.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/3321/files#diff-46d00b49fb68d6da13c9ecdc5ab3dab6adf502ec216349721586e96553652444R246) ✅:

https://github.com/matter-labs/zksync-era/blob/facb415f3c3ef7f5cc3d423963349a237bb4a5e5/core/lib/dal/src/tee_proof_generation_dal.rs#L241-L281